src/app/features/exchange-set/ess-upload-file/ess-upload-file/ess-upload-file.component.ts

```diff
@@ -14,7 +14,7 @@ export class EssUploadFileComponent implements OnInit {
   displayErrorMessage = false;
   validEncList: string[];
   encFile: File;
-  // public displayedColumns = ['Cell name', 'Choose'];
+  private sendErrorMessage: any;
```

Comment:
Please delete `sendErrorMessage` member if not already done when resolving merge conflicts